### PR TITLE
Added explanation for find centroid algorithm

### PR DESCRIPTION
**Pull request by jeffarjeffar**
I think this makes it easier to understand. 
You guys never provided an explanation for how to find the centroid, and jumped directly to the centroid decomp algorithm.

### CODE REVIEW

File: content/5_Plat/Centroid.mdx [hunk 1]
```diff
@@ -17,11 +17,77 @@ TODO:
...
 <FocusProblem problem="sam" />
 
-**Centroid Decomposition** is a divide and conquer technique for trees. The
-**centroid** of a tree is a node which, if rooted, results in no other nodes
-having a subtree of size greater than $\frac{N}{2}$. **Centroid Decomposition**
+<LanguageSection>
+
+<CPPSection>
+
+We can find a centroid in a tree by starting at the root. Each step, loop through all of it's children. If all of it's children have subtree size less than or equal to $\frac{N}{2}$, then it is a centroid. Otherwise, move to the child whose subtree is more than $\frac{N}{2}$ and repeat until you find a centroid.
```
Comment (bqi343) [inline content/5_Plat/Centroid.mdx]: its

Comment (bqi343) [reply content/5_Plat/Centroid.mdx]: > therwise, move to the child whose subtree is more than $\frac{N}{2}$

subtree size

File: content/5_Plat/Centroid.mdx [hunk 2]
```diff
@@ -17,11 +17,77 @@ TODO:
 
 ## Introduction
 
+### Centroids
+
+The **centroid** of a tree is a node which, if rooted, results in no other nodes
```
Comment (bqi343) [inline content/5_Plat/Centroid.mdx]: I think the following is better?

> a node such that rooting the tree at that node results in no other ...

**Review by bqi343 — CHANGES_REQUESTED** [review]

File: content/5_Plat/Centroid.mdx [hunk 3]
```diff
@@ -17,13 +17,85 @@ TODO:
...
-works by repeated splitting the tree and each of the resulting subgraphs at the
-centroid, producing $\mathcal{O}(\log N)$ layers of subgraphs.
+<LanguageSection>
+
+<CPPSection>
+
+We can find a centroid in a tree by starting at the root. Each step, loop
+through all of its children. If all of its children have subtree size less
+than or equal to $\frac{N}{2}$, then it is a centroid. Otherwise, move to the
+child whose subtree is more than $\frac{N}{2}$ and repeat until you find a
```
Comment (bqi343) [inline content/5_Plat/Centroid.mdx:34]: > whose subtree is more than $\frac{N}{2}$

subtree size is greater than $\frac{N}{2}$?

also pls re-request review next time

**Review by bqi343 — CHANGES_REQUESTED** [review]

**Review by bqi343 — APPROVED** [review]

File: content/5_Plat/Centroid.mdx [hunk 4]
```diff
@@ -17,13 +17,85 @@ TODO:
...
-**Centroid Decomposition** is a divide and conquer technique for trees. The
-**centroid** of a tree is a node which, if rooted, results in no other nodes
-having a subtree of size greater than $\frac{N}{2}$. **Centroid Decomposition**
-works by repeated splitting the tree and each of the resulting subgraphs at the
-centroid, producing $\mathcal{O}(\log N)$ layers of subgraphs.
+<LanguageSection>
+
+<CPPSection>
+
+We can find a centroid in a tree by starting at the root. Each step, loop
```
Comment (dolphingarlic) [inline content/5_Plat/Centroid.mdx:31]: Move this paragraph to outside the LanguageSection

**Review by dolphingarlic — CHANGES_REQUESTED** [review]

**Pull request merged**